HWLesson6_8/main.js

```diff
@@ -0,0 +1,137 @@
+        <p>Technologie: ${RowentaCln.optional.technology}</p>
+        <p><i><b>${RowentaCln.price} €</p>
+        `;
+
+cleanerListEl2.innerHTML = `<h3>${DysonCln.name}</h3>
+        <p>Rating: ${DysonCln.rating}</p>
+        <p>Capacität: ${DysonCln.optional.liter} L</p>
+        <p>Faber: ${DysonCln.optional.color}</p>
+        <p>Quelle: ${DysonCln.optional.electricSource}</p>
+        <p>Technologie: ${DysonCln.optional.technology}</p>
+        <p><i><b>${DysonCln.price} €</p>
+        `;
+
+cleanerListEl3.innerHTML = `<h3>${AegCln.name}</h3>
+        <p>Rating: ${AegCln.rating}</p>
+        <p>Capacität: ${AegCln.optional.liter} L</p>
+        <p>Faber: ${AegCln.optional.color}</p>
+        <p>Quelle: ${AegCln.optional.electricSource}</p>
+        <p>Technologie: ${AegCln.optional.technology}</p>
+        <p><i><b>${AegCln.price} €</p>
+        `;
+
+// Add products to shopping cart
+        //Get button's element
+const addBoschButton = document.getElementById("addBoschToCart");
+const addRowentaButton = document.getElementById("addRowentaToCart");
+const addDysonButton = document.getElementById("addDysonToCart");
+const addAegButton = document.getElementById("addAegToCart");
+const showShoppingCartButton = document.getElementById("showShoppingCart");
+const showTotalPriceButton = document.getElementById("totalPrice");
+
+        //create shoppingCart variable to store each product
+    const shoppingCart = [];
+
+        //function to add products to shoppingCart and update the quantity
+    function addBoschToShoppingCart() {
+        const product = {
+            name:"Bosch BGC05A220A",
+            price: 125
+        }
+        shoppingCart.push(product);
+        console.log("shoppingCart: ", shoppingCart);
+    }
+
+    function addRowentaToShoppingCart() {
+        const product = {
+            name:"Rowenta RO 3731EA Compact Power Cyclonic",
+            price: 145
+        }
+        shoppingCart.push(product);
+        console.log("shoppingCart: ", shoppingCart);
+    }
+
+    function addDysonToShoppingCart() {
+        const product = {
+            name:"Dyson Cyclone V10 Animal (226379-01)",
+            price: 450
+        }
+        shoppingCart.push(product);
+        console.log("shoppingCart: ", shoppingCart);
+    }
+
+    function addAegToShoppingCart() {
+        const product = {
+            name:"AEG CX7-2-S360 X Flexibility",
+            price: 385
+        }
+        shoppingCart.push(product);
+        console.log("shoppingCart: ", shoppingCart);
+    }
+
+    //function to display showShoppingCartProducts
+    function showShoppingCartProducts() {
+        const shoppingCartContainer = document.getElementById("shoppingCart");
+        shoppingCartContainer.innerText = '';
+        for(let i = 0; i < shoppingCart.length; i++) {
+            console.log(shoppingCart[i]);
+            const productContainer = document.createElement('div')           

```

Comment:
You can use more styles to improve you css skills.

Reply:
Thank you very much for your detailed feedbacks, I will try to add more features to my website. I have a question: I have invited 5 teachers from the beginning and also merged the pull request. But they seem not to see my code and still awaiting to review. Could you please help me out what to solve this problem?
Thank you!

Reply:
@HoangMinhTuan-ops could you please revert the commit. Since this pr was not approved by any teacher.